packages/account/src/Sections/Verification/ProofOfAddress/__tests__/proof-of-address.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
```suggestion
```

Reply:
POA is used inside CFD package as well. So I cant remove it